### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by Owusu-Desmond**
In this pull request, I implement the following

- [ ] I use controllers to handle requests and render some views.
- [ ] I make sure that I added code that handles only the 4 required URLs.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+    puts @users
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+end
```
Comment (Owusu-Desmond) [inline app/controllers/users_controller.rb:10]: gdgdfgjdfkgfhgkdfg

File: config/routes.rb [hunk 2]
```diff
@@ -2,5 +2,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+
+  resources :users, only: [:index, :show] do
```
Comment (Owusu-Desmond) [inline config/routes.rb:7]: dlkfhdskfdhsf

**Review by Owusu-Desmond — COMMENTED** [review]
Hi

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Owusu-Desmond, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**